src/learning_item_data_retriever.py

```diff
@@ -0,0 +1,75 @@
+import json
+import pandas as pd
+from learning_item import LearningItem
+
+FOLDER_PATH = "E:\\Mi unidad\\_PYTHON\\PycharmProjects\\memoryBooster\\data\\"
```

Comment:
If someone else will try to run this application this variables will change. There can be a misunderstanding if someone will push them to repository. Use enviroment variables instead to assign to those final variables. And add a note about what to add in readme.md of the application

Reply:
Just to be sure. 
You mean about such change:
os.environ['FOLDER_PATH '] = 'value' 
set up previously in oparating system?

Reply:
Done - please review it